pkg/spec/populators.go

```diff
@@ -157,13 +157,13 @@ func populateContainers(containers []Container, cms []ConfigMapMod, secrets []Se
 		// process health field
 		c, err := populateProbes(c)
 		if err != nil {
-			return cnts, errors.Wrapf(err, "error converting 'health' to 'probes', app.containers[%d]", cn)
+			return cnts, errors.Wrapf(err, "error converting 'health' to 'probes', deployment.containers[%d]", cn)
```

Comment:
This will still be `app`, because of the following reasons:

- this is not tied to deployment in anyway
- user will still be considering it as app
- the error is user facing

Reply:
ack, changed